src/compile/compile.ts

```diff
@@ -48,7 +48,7 @@ export function compileRootGroup(model: Model) {
   const spec = model.spec();
 
   let rootGroup:any = extend({
-      name: spec.name ? spec.name + '-root' : 'root',
+      name: spec.name ? spec.name + '-rootCell' : 'rootCell',
```

Comment:
Oh I see. I assumed that what is currently Vega-Lite's "root" group mark would become a unit cell. But I could also see introducing an additional group cell for each unit. Feel free to change as you see fit! No other reserved names that I can think of.